includes/class-indieauth-authenticate.php

```diff
@@ -114,14 +114,17 @@ public function determine_current_user( $user_id ) {
 
 	}
 
-	public function verify_access_token( $token ) {
+	public function verify_access_token( $token, $endpoint = null ) {
+		if ( ! $endpoint ) {
+			$endpoint = get_option( 'indieauth_token_endpoint', rest_url( 'indieauth/1.0/token' ) );
```

Comment:
this fallback does not work, you never want to use the local endpoint to verify!